### ISSUE TITLE
duplicate account entries across extensions are causing breakages

### ISSUE THREAD

**Issue by wirednkod** [paritytech/polkadot-staking-dashboard#378]
In "Nominate", menu the component `Nominations` never stops syncing and/or loading the nominations from the account:

![image](https://user-images.githubusercontent.com/5408605/203572845-62ea4f22-c37a-4fc0-bd46-257d593d6efd.png)

**Comment by rossbulat** [comment]
Thanks for reporting, does this happen with the latest master branch? Just would like to confirm it is using the latest dependency bumps before we continue to investigate.

**Comment by wirednkod** [comment]
> Thanks for reporting, does this happen with the latest master branch? Just would like to confirm it is using the latest dependency bumps before we continue to investigate.

This is in the live one **and** in the latest master branch

**Comment by rossbulat** [comment]
I am unable to reproduce. I am not sure what is causing this.

**Comment by wirednkod** [comment]
I just realised there is an error in the console multiple times :
```
Error: Unable to handle message of type pub(accounts.unsubscribe)
    at it.origin.handleResponse (page.js:2:91803)
    at page.js:2:92055
```

**Comment by wirednkod** [comment]
Once deactivating Talisman extension everything went back to normal. Will close this as it is irrelevant to Polkadot Staking Dashboard

**Comment by wirednkod** [comment]
After some more testing I realised that this appears when same account exists in both polkadotJS extension and Talisman extension - plus it is irrelevant to the error mentioned. 
I think that there is some conflict when both more than 1 extensions have the same account

**Comment by rossbulat** [comment]
Fixed in #391
